assets/src/utils/charts.ts

```diff
@@ -3,19 +3,26 @@
 import uPlot, { AlignedData } from "uplot";
 import { getXTicksConfig } from "../utils/chartsXTicksConfig";
 
+const metricToTitle: { [metric: string]: string } = {
+  caps: "Processed caps",
+  event: "Processed events",
+  store: "Input buffer size inside store/3",
+  take_and_demand: "Input buffer size inside take_and_demand/4",
+};
+
 export function createCharts(
   container: HTMLElement,
   width: number,
-  method: string
+  metric: string
 ): uPlot {
   const data: AlignedData = [[]];
 
   return new uPlot(
     {
       width: width,
       height: 200,
-      title: `Input buffer size inside ${method}`,
-      id: "chart1",
+      title: metricToTitle[metric] ?? "metric",
```

Comment:
I don't know if I remember how it looked before the CR suggestion but are you sure that you want the title to be `"metric"` instead of `metric`?

Reply:
It should be `metric`.